Apps/Playground/playground-shared/App.tsx

```diff
@@ -140,7 +140,7 @@ const EngineScreen: FunctionComponent<ViewProps> = (props: ViewProps) => {
                 <Image style={{flex: 1}} source={{uri: snapshotData }} />
               </View>
             }
-            <EngineView camera={camera} onInitialized={onInitialized} displayFrameRate={true} antiAliasing={2}/>
```

Comment:
Can you try with this enabled again? I think @CedricGuillemet made a fix for this in BGFX a few days ago, and I think that has made it's way to master in BRN as well.

Reply:
Verified this works now